sdk/identity/Azure.Identity/src/Policies/DefaultAzureCredentialImdsRetryPolicy.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Identity
+{
+    internal class DefaultAzureCredentialImdsRetryPolicy : RetryPolicy
+    {
+        public DefaultAzureCredentialImdsRetryPolicy(RetryOptions retryOptions, DelayStrategy delayStrategy = null) : base(retryOptions.MaxRetries,
+                    delayStrategy ?? DelayStrategy.CreateExponentialDelayStrategy(retryOptions.Delay, retryOptions.MaxDelay))
```

Comment:
Out of curiosity, was this the scheme all languages decided to go with? I remember from conversations the probing delays/retries/etc. were all over the place across languages.

Reply:
This logic is only specific to whether we do retries or not. The specific delay sequence could also be implemented here, but for now it is just the default exponential strategy.